src/finch/resources/hris/pay_statements.py

```diff
@@ -46,7 +46,7 @@ def retrieve_many(
         Read detailed pay statements for each individual.
 
         Deduction and contribution types are supported by the payroll systems that
-        support Benefits.
+        supports Benefits.
```

Comment:
@arditP is this a change we need to make in the API spec? The previous change is correct in this case